programs/vote_api/src/vote_instruction.rs

```diff
@@ -52,64 +52,81 @@ impl std::error::Error for VoteError {}
 pub enum VoteInstruction {
     /// Initialize the VoteState for this `vote account`
     /// takes a node_pubkey and commission
-    InitializeAccount(Pubkey, u8),
+    InitializeAccount(Pubkey, Pubkey, Pubkey, u8),
```

Comment:
Make this a struct instead of a tuple, so the Pubkeys are easier to distinguish?